ui/src/hooks/useActions.ts

```diff
@@ -116,11 +118,17 @@ const mergeData = (
       timestamp: wei(futuresTrade.timestamp),
       size: wei(futuresTrade.size, 18, true),
       leverage: null, // todo add leverage
+      sizeInUsd: Math.abs(
```

Comment:
IIRC we have an entry and exit price on the graph - do we want to distingush between those? And maybe liquidation price if it is a liquidation...